### ISSUE TITLE
Authentication doesn't work any more

### ISSUE THREAD

**Issue by Bronkoknorb** [vinitkumar/googlecl#250]
When I try to authenticate I get the following message:

Important: OAuth 1.0 was officially deprecated on April 20, 2012, and is no longer supported. We encourage you to migrate to OAuth 2.0 as soon as possible.

**Comment by vinitkumar** [comment]
@Bronkoknorb Thanks for the issue, I am on vacation this and next week. Will migrate to OAuth 2.0 when I am back.

**Comment by paulfitz** [comment]
looks like upstream may also be trying to fix this, see https://code.google.com/p/googlecl/issues/detail?id=573#c22

> "I spent a few hours trying to upgrade and didn't have much success.  Talking to the oauth folks now, but no promises."

**Comment by vinitkumar** [comment]
@paulfitz Thanks, Looking into it.

**Comment by vinitkumar** [comment]
- Note to self : https://developers.google.com/api-client-library/python/guide/aaa_oauth

**Comment by giacecco** [comment]
Don't exclude the option to use "app passwords" rather than OAuth. Some applications may not need need that secure.

**Comment by vinitkumar** [comment]
@giacecco Alright, Will keep that in mind.

**Comment by xiconet** [comment]
Hi,
Before all, thanks for all your work.
It seems that ``googlecl`` is now in a state of abandon: major "bug" still not fixed more than one year later.
Please correct me if I'm wrong
Best regards

**Comment by vinitkumar** [comment]
@xiconet On the contrary, I have recently regained work on refactoring the authentication part. It is going good but gdata (the library to talk to google services) has major breaking changes and I need to change some api interaction part. This is a major refactor and will take some time. Though, I have decided to port it one by one. I am planning to first upgrade the google docs part so that googlecl partially works and with time and support for community hopefully we will get it fully functional.
